hack/update-codegen.sh

```diff
@@ -26,7 +26,7 @@ readonly GOMODCACHE="$(go env GOMODCACHE)"
 readonly GO111MODULE="on"
 readonly GOFLAGS="-mod=readonly"
 readonly GOPATH="$(mktemp -d)"
-readonly MIN_REQUIRED_GO_VER="1.20"
+readonly MIN_REQUIRED_GO_VER="$(go list -m -f '{{.GoVersion}}')"
```

Comment:
picks up go version from the go.mod file

Reply:
Neat, thanks!